.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "airbnb",
+    "airbnb-typescript"
+  ],
+  "overrides": [],
+  "parserOptions": {
+    "extends": [
+      "airbnb",
+      "airbnb-typescript"
+    ],
+    "ecmaVersion": "latest",
+    "sourceType": "module",
+    "project": [
+      "./tsconfig.json"
+    ]
+  },
+  "plugins": [
+    "react"
+  ],
+  "rules": {
+    "react/function-component-definition": [
+      2,
+      {
+        "namedComponents": [
+          "function-declaration",
```

Comment:
I would recommend to use one approach of function declaration to be more consistent

Reply:
It's just recommendation - let's try to setup at least rules for code styling such as line-breaks, extra-spaces etc